components/ThePlayground.client.vue

```diff
@@ -14,6 +13,12 @@ const isDragging = usePanelDragging()
 const panelSizeEditor = usePanelCookie('nuxt-playground-panel-editor', 30)
 const panelSizeFrame = usePanelCookie('nuxt-playground-panel-frame', 30)
 
+const { location, wcUrl } = usePlayground()
```

Comment:
Maybe it's better not to name it `location` in case to swallows the global `location`?

Reply:
oh totally agree, let me change it to `iframeLocation`, feel free to rename it if you got a better idea.